lib/bump/bump.ts

```diff
@@ -83,6 +110,29 @@ export interface AutoBumpProps extends AutoBumpOptions {
   repo: WritableGitHubRepo;
 }
 
+export interface PullRequestOptions {
+  /**
+   * The title of the pull request.
+   *
+   * $VERSION will be substituted by the current version (obtained by executing `versionCommand`).
+   *
+   * @default "chore(release): $VERSION"
+   */
+  title?: string;
+
+  /**
+   * The PR body.
+   * @default "see CHANGELOG"
+   */
+  body?: string;
```

Comment:
Given that the most important part of the human verification here would be verifying the CHANGELOG, I would prefer to see it pasted directly in the PR body so we can see at a glance whether it is formatted correctly and doesn't contain strange items.

Reply:
I will leave it for a subsequent PR. It might be a bit tricky to support this at the delivlib level because we are not opinionated about changelogs here.